packages/api/amplify_api/example/.gitignore

```diff
@@ -49,7 +49,6 @@ amplify/.config/local-*
 amplify/logs
 amplify/mock-data
 amplify/backend/amplify-meta.json
-amplify/backend/awscloudformation
```

Comment:
Why is this being removed?

Reply:
Probably a CLI update - they change these values sometimes